### PR TITLE
BUG: to_latex() output broken when the index has a name (GH10660)

### LINKED ISSUE #10660: BUG: to_latex() output broken when the index has a name
Hey folks,

I posted this on [SO](https://stackoverflow.com/questions/31582114/groupby-to-latex-output-broken) and was asked to file a report here as well. 

I'm trying to export `pandas.DataFrame.describe()` to `LaTex` using the `to_latex()`-method. This works all fine as long as I don't apply the `groupby()`-method beforehand. With a grouped DataFrame, the first row has no values, even though its label is `count`. Note that the first row of a grouped dataframe is used to mark down the variable used for grouping in iPython notebook.

I'm using pandas 0.16.2, python 3.
Is this a bug or am I doing something wrong? 

Cheers,
Jakob

Here some examples:

Without `groupby`:

    \begin{tabular}{lr}
    \toprule
    {} &    IS\_FEMALE \\
    \midrule
    count &  2267.000000 \\
    mean  &     0.384649 \\
    ...
    ...
    75\%   &     1.000000 \\
    max   &     1.000000 \\
    \bottomrule
    \end{tabular}

[![enter image description here][1]][2]

With `groupby`:

    \begin{tabular}{llr}
    \toprule
      &       &    IS\_FEMALE \\
    \midrule
    0 & count &              \\     % <-- note missing value here
      & mean &  1134.000000 \\
      & std &     0.554674 \\
    ...
    ...
      & 75\% &     0.000000 \\
      & max &     0.000000 \\
    \bottomrule
    \end{tabular}

[![enter image description here][3]][4]

Output in the notebook:

[![enter image description here][5]][5]

  [1]: http://i.stack.imgur.com/gK10V.png
  [2]: http://i.stack.imgur.com/gK10V.png
  [3]: http://i.stack.imgur.com/lt9z9.png
  [4]: http://i.stack.imgur.com/lt9z9.png
  [5]: http://i.stack.imgur.com/wT8Tp.png

### PR DESCRIPTION
**Pull request by taeold**
closes #10660

first try at contributing to pandas. feedbacks will be greatly appreciated.

### CODE REVIEW

File: pandas/tests/test_format.py [hunk 1]
```diff
@@ -2648,6 +2648,23 @@ def test_to_latex_multiindex(self):
 """
         self.assertEqual(result, expected)
 
+        df = pd.DataFrame({'a':[0,0,1,1], 'b':list('abab'), 'c':[1,2,3,4]})
+        df = df.set_index(['a', 'b'])
```
Comment (jreback) [inline pandas/tests/test_format.py]: can you also provide a test using the groupby in the issue

Comment (taeold) [inline pandas/tests/test_format.py]: done!

**Comment by jreback** [comment]
looks good. pls squash.

File: pandas/tests/test_format.py [hunk 2]
```diff
@@ -2648,6 +2648,49 @@ def test_to_latex_multiindex(self):
 """
         self.assertEqual(result, expected)
 
+        df = pd.DataFrame({'a':[0,0,1,1], 'b':list('abab'), 'c':[1,2,3,4]})
```
Comment (jreback) [inline pandas/tests/test_format.py]: add the issue number here as a comment

Comment (taeold) [inline pandas/tests/test_format.py]: will do - thanks for the review

**Comment by jreback** [comment]
thank you sir!

**Pull request merged**